src/Build/Logging/BinaryLogger/Postprocessing/StreamExtensions.cs

```diff
@@ -60,7 +60,9 @@ public static byte[] ReadToEnd(this Stream stream)
         {
             if (stream.TryGetLength(out long length))
             {
-                BinaryReader reader = new(stream);
+                // check with Jan
```

Comment:
This looks fine to me (but mostly I'm just putting a comment here so this is more visible)